docs/_static/language-support.html

```diff
@@ -103,6 +103,13 @@
             <td style="background-color:#dff0d8; padding: 5px;"><a href="../advanced-topics/best-practices/nfl-best-practices.html#best-practices">Yes</a></td>
             <td style="background-color:#dff0d8; padding: 5px;">Yes</td>
         </tr>
+        <tr style="border-top: 1px solid #ddd; ">
+            <td style="padding: 5px;"><a href="../faq.html#what-is-an-open-tool-or-workflow">Open Data</a></td>
+            <td style="background-color:#dff0d8; padding: 5px;">Yes <sup>11</sup></td>
+            <td style="background-color:#dff0d8; padding: 5px;">Yes<sup>12</sup></td>
```

Comment:
Minor nit, but the other rows put the numbers in square brackets like `11` -> `[11]`